src/components/Lesson/ImprovePage.scss

```diff
@@ -81,25 +38,34 @@
 }
 
 /* Student / Teacher mode */
+
+/* Box shadow inset used to highlight box so it differentiates better from other objects */
 .improvePageBoxStudent {
   composes: improvePageBox;
   background-color: $color-green-light;
+  box-shadow: inset 0 0 85px #89c785;
 }
 
+/* Works as a <hr/> to differentiate text and links */
+.improvePageBoxStudent .improvePageBoxTextRow {
```

Comment:
Hmm, hva mener du med punkt 2/3? :p

- Jeg endret classenavnene til mer leselige navn, 
- ".student .textRow" og ".teacher .textRow" linjene må vel nesten være sann siden box shadow-en er forskjellge farger

- tror jeg velger å beholde composes akkurat her, føler det er ganske oversiktlig (men skjønner godt hva du mener med at det kan bli for 'rotete')